### ISSUE TITLE
Is our world virtual?

### ISSUE THREAD

**Issue by zzp-coder** [pose-uob/POSE2022#37]

**Comment by YidaGuo** [comment]
No, I’m sure life is real because we can feel things around us. But what I do know for sure is that, it’s pitch black after death. It’ll be like you went to sleep and never woke up.

**Comment by zzp-coder** [comment]
Your comment is reasonable,but i think roles of computers games can feel things around them,too.Hence, i think we can't find true evidence to prove that our world is not virtual.
